tmail-backend/jmap/extensions-es/src/test/java/com/linagora.tmail.james.jmap.model/ESEmailAddressContactSearchTest.java

```diff
@@ -1,12 +1,11 @@
 package com.linagora.tmail.james.jmap.model;
 
 import com.linagora.tmail.james.jmap.ESEmailAddressContactSearchEngine;
-import com.linagora.tmail.james.jmap.model.EmailAddressContactSearchEngine;
 import org.apache.james.backends.es.v7.DockerElasticSearchExtension;
 import org.apache.james.backends.es.v7.ReactorElasticSearchClient;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-public class ESEmailAdressContactSearchTest implements EmailAddressContactSearchEngineContract{
+public class ESEmailAddressContactSearchTest implements EmailAddressContactSearchEngineContract{
```

Comment:
Unrelated changr

Reply:
May I just put it here, as suddenly found out that dictation error so couldn't help editing :(

Reply:
You should change it in the related PR or separated one, or separated commit. This has nothing to do with the mailet :)